src/styles/calendar.scss

```
@@ -0,0 +1,66 @@
+$size: 100px;
+$gap: 1px;
+$calendar-width: calc(7 * #{$size} + 6 * #{$gap});
+
+@mixin square($size) {
+  height: $size;
+  width: $size;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: 1px;
+
+  $day-of-week:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $name, $day in $day-of-week {
+    .calendar--start-day_#{$name} &:first-child {
```

```suggestion
    .calendar--start-day_#{$name} .calender__day:first-child {
```
start day modifier also works incorrect.
loks like incorrect selector. try something like that